plugin.rb

```diff
@@ -872,6 +872,15 @@ module ::DiscourseAssign
     end
   end
 
+  on(:group_destroyed) do |group, user_ids|
+    user_ids.each do |user_id|
+      user = User.find(user_id)
```

Comment:
```suggestion
    User.where(id: user_ids).find_each do |user|
```

Seems much more efficient to find all the users at once, and in batches in case the group has tons of users.